Android/app/src/main/java/fashiome/android/adapters/ProductAdapter.java

```diff
@@ -213,9 +213,9 @@ public void configure(Product product) {
             tvProductTitle.setText(product.getProductName());
             tvProductDescription.setText(product.getProductDescription());
 
-//            if ( product.getProductPostedBy().getFullName() != null) {
-//                tvProductByUserName.setText( product.getProductPostedBy().getFullName());
-//            }
+            if (product.getProductPostedBy().getUsername() != null) {
+                tvProductByUserName.setText(product.getProductPostedBy().getUsername());
```

Comment:
You can anything now .. Problem is from FB we get fullname we store as username only